### ISSUE TITLE
Dependency documentation is missing things

### ISSUE THREAD

**Issue by aMOPel** [h3rald/hastyscribe#88]
I needed to 
`nimble install nifty`
and
`sudo apt install libmarkdown2-dev`
before
`nimble install hastyscribe`
 would work without error.

Maybe you want to add those to the installation instructions, and/or improve on the `.nimble` file to install nifty automatically before calling it.

Thanks for you work :)

**Comment by Diogenesoftoronto** [comment]
yeah ran into the same problem. I am going to make a pr with those add notes

**Comment by h3rald** [comment]
Simplified the installation in v2.0.0
